src/ReleaseHistory.md

```diff
@@ -0,0 +1,5 @@
+# Microsoft Json Schema Packages
+
+## **0.57.0** [Pointer](https://www.nuget.org/packages/Microsoft.Json.Pointer/0.57.0) | [Schema](https://www.nuget.org/packages/Microsoft.Json.Schema/0.57.0)| [Schema.ToDotNet](https://www.nuget.org/packages/Microsoft.Json.Schema.ToDotNet/0.57.0)| [Schema.Validation](https://www.nuget.org/packages/Microsoft.Json.Schema.Validation/0.57.0)
+* Remove dependency on result.ruleMessageId
```

Comment:
>Remove dependency on result.ruleMessageId [](start = 2, length = 41)

How about "The SARIF log file produced by the validation tool no longer uses the obsolete SARIF property `result.ruleMessageId`.